File: Python Word Examples/BackgroundApiDemo/setBackgroundImageInRequest.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.background_api.BackgroundApi(configuration)

filename = 'SetBackgroundImageInRequest.docx'
folder = 'input'
imgePath = 'D:/input/Logo.png'
dest_file_path = 'output/SetBackgroundImageInRequest.docx'

result = api.set_background_image_in_request(name=filename, folder=folder, input_image=imgePath,
                                             dest_file_path=dest_file_path)

File: PHP Word Examples/BackgroundApiDemo/setBackgroundImageInRequest.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\BackgroundApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new BackgroundApi($configuration);

$name = "setBackgroundImageWithImgFile.docx";
$input_image = "D:/data/Background.png";
$password = null;
$folder = "input";
$storage = null;
$dest_file_path = "output/setBackgroundImageWithImgFile.docx";
$result=$apiInstance->setBackgroundImageInRequest($name, $input_image,$dest_file_path, $folder, $storage, $password);